client-encryption/src/main/java/io/confluent/kafka/schemaregistry/encryption/local/LocalKmsClient.java

```diff
@@ -63,25 +62,24 @@ private LocalKmsClient(String uri, String secret, List<String> oldSecrets)
     }
     this.keyUri = uri;
 
-    KeyManager<Aead> keyManager = Registry.getKeyManager(
-        "type.googleapis.com/google.crypto.tink.AesGcmKey", Aead.class);
     PrimitiveSet.Builder<Aead> builder = PrimitiveSet.newBuilder(Aead.class);
-    builder.addPrimaryPrimitive(getPrimitive(keyManager, secret), getKey(secret));
+    builder.addPrimaryPrimitive(getPrimitive(secret), getKey(secret));
     for (String oldSecret : oldSecrets) {
-      builder.addPrimitive(getPrimitive(keyManager, oldSecret), getKey(oldSecret));
+      builder.addPrimitive(getPrimitive(oldSecret), getKey(oldSecret));
     }
     this.aead = Registry.wrap(builder.build());
   }
 
-  private Aead getPrimitive(KeyManager<Aead> keyManager, String secret)
+  private Aead getPrimitive(String secret)
       throws GeneralSecurityException {
     byte[] keyBytes = Hkdf.computeHkdf(
         "HmacSha256", secret.getBytes(StandardCharsets.UTF_8), null, null, 16);
     AesGcmKey key = AesGcmKey.newBuilder()
         .setVersion(0)
         .setKeyValue(ByteString.copyFrom(keyBytes))
         .build();
-    return keyManager.getPrimitive(key);
+    return Registry.getPrimitive(
+        "type.googleapis.com/google.crypto.tink.AesGcmKey", key.toByteString(), Aead.class);
```

Comment:
Can we define a const for this if there isn't one already?

Reply:
Done